templates/bootstrap/plugin_name/app/models.py.j2

```diff
@@ -70,5 +70,7 @@ class {{ plugin_camel_short }}Repository(Repository):
 
     TYPE = "{{ plugin_dash_short }}"
 
+    CONTENT_TYPES = [{{ plugin_camel_short }}Content]
```

Comment:
New addition, need to document this for the GA.  Plan to use this soon since @fabricio-aguiar was able to verify that we have a large memory consumption issue with duplicate resolution, and this is a very straightforwards and efficient way to simplify underlying layers (and simultaneously enable some useful extra validation).

See: https://pulp.plan.io/issues/5701

Reply:
I'd like to see the other PRs that correspond with this. I don't think we want to add to the plugin template before adding it to the code itself. That would prevent this from merging, so maybe more this to its own PR?

Reply:
My concerns on this thread are resolved by seeing the other PR. Thank you.